ADOPTERS.md

```diff
@@ -0,0 +1,18 @@
+# GrimoireLab Adopters
+
+This is the list of organizations and users that have publicly shared how
+they are using GrimoireLab.
+
+💡 **Add your organization by creating a PR**
+
+The list of organizations that have publicly shared the usage of GrimoireLab:
+
+| Organization                      | Success Story                                                                                                                                  |
+|:----------------------------------|:-----------------------------------------------------------------------------------------------------------------------------------------------|
+| [Bitergia](https://bitergia.com/) | We provide a ready-to-use platform - built on top of GrimoireLab - to empower managers to reach their business goals with Open Source metrics. |
```

Comment:
Proposed alternative that is more inclusive of the different business lines Bitergia has.

```suggestion
| [Bitergia](https://bitergia.com/) | We offer a commercial ready-to-use platform, built with GrimoireLab, that gives managers software development analytics for the projects they care about. |
```